src/draive/splitters/basic.py

```diff
@@ -33,28 +33,32 @@ def split_text(
 
 def _split(
     text: str,
-    separators: tuple[str, str] | str | None = None,
+    separators: Sequence[str] | str | None = None,
 ) -> tuple[str, list[str]]:
```

Comment:
we should return reminder splitters from the list I believe - in merge we should further try on those - we can discuss that thing